peers/open_channels.js

```diff
@@ -817,6 +784,16 @@ module.exports = (args, cbk) => {
           return cbk();
         }
 
+        // Exit early if avoiding broadcast
```

Comment:
This should be moved down just before the broadcast because it wouldn't be safe for the caller to self-publish this transaction yet